Ability to set Window Flags

meir opened the pull request:
**Changes**
- Added `WindowFlag` enum
- Changes call from `(app *App) NewWindow(uint, uint, bool, string) *Window` to `(app *App) NewWindow(uint, uint, bool, string, WindowFlag) *Window`
- Replaced default flags with flags given to a window to the WindowFlag variable given with the NewWindow call

**Example**
```go
package main

import (
	"github.com/raff/ultralight-go"
)

func main() {
	app := ultralight.NewApp()
	defer app.Destroy()

	window := app.NewWindow(800, 600, false, "Window", ultralight.WindowFlagBorderless | ultralight.WindowFlagResizable)
	defer window.Destroy()

	window.View().LoadURL("file:///assets/index.html")

	window.Focus()
	app.Run()
}
```

raff commented:
Maybe it would be better to add a new constructor (NewWindowWithFlag) or have a DefaultWindowFlags constant set to the current default flags.

meir commented:
I've reverted the `NewWindow` method back to its original state and added the method `NewWindowUsingFlags`
I have also added an example for the flags as `examples/flags.go`
